assets/src/scss/base/_typography.scss

```diff
@@ -25,11 +25,6 @@ h3,
 h4,
 h5,
 h6 {
-  --headings-- {
-    font-family: $roboto;
-    font-weight: bold;
-  }
```

Comment:
I see [some code using these variables](https://github.com/greenpeace/planet4-child-theme-indonesia/blob/8ac284d4adc714447502c8c119dd0b5bdbca486f/assets/src/scss/layout/_page-section.scss#L4-L5), I guess it won't have an impact because of default values but we could warn them (or warn the dev channel in general)